specs/experimental/gov-proposal-validator.md

```diff
@@ -51,67 +51,157 @@ The contract has a single `owner` role (Optimism Foundation) with permissions to
 
 ### Public Functions
 
-`submitProposal`
+`submitProtocolOrGovernorUpgradeProposal`
```

Comment:
mention that it also MUST store the timestamp of submission for all submit functions